lib/ansible/modules/storage/netapp/sf_account_manager.py

```diff
@@ -0,0 +1,284 @@
+#!/usr/bin/python
+# (c) 2016, NetApp, Inc
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+#!/usr/bin/python
+
+DOCUMENTATION = '''
+
+module: sf_account_manager
+
+short_description: Manage SolidFire accounts
+version_added: '2.3'
+author: Sumit Kumar (sumit4@netapp.com)
+description:
+- Create, destroy, or update accounts on SolidFire
+    - auth_basic
+
+options:
+
+    hostname:
+        required: true
+        description:
+        - hostname
+
+    username:
+        required: true
+        description:
+        - username
+
+    password:
+        required: true
+        description:
+        - password
+
+    action:
```

Comment:
Parameter of **action** is against best practices for module dev.  Please use **state** in place of this.